numpy/core/src/multiarray/array_method.h

```diff
@@ -104,6 +104,28 @@ typedef int (get_loop_function)(
         NPY_ARRAYMETHOD_FLAGS *flags);
 
 
+/**
+ * Query an ArrayMethod for the initial value for use in reduction.
+ *
+ * @param context The arraymethod context, mainly to access the descriptors.
+ * @param reduction_is_empty Whether the reduction is empty. When it is, the
+ *     default value for the identity might differ, for example:
```

Comment:
```suggestion
 *     initial value for the identity might differ, for example:
```
The function returns the "initial" value.  The term "default" (for the value returned by the empty reduction) should only be used in the bullet points later.  

I don't hate the term "default", but we don't use it anywhere.  So I think Matti is right and I should rephrase this to, at least mainly, just not use it.